services/QuillLMS/spec/controllers/api/v1/activity_questions_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::ActivityQuestionsController, type: :controller do
+  let(:redis) { Redis.new }
+
+  before do
+    allow(Rails).to receive(:cache).and_return(ActiveSupport::Cache::RedisCacheStore.new(redis: redis))
+  end
+
+  describe "GET #index" do
+    let(:activity) { create(:activity) }
+    let(:questions) { create_list(:question, 3) }
+    let(:locale) { 'en' }
+    let(:cache_key) { "activity_questions/#{activity.uid}/#{locale}" }
+
+    before do
+      allow(Activity).to receive(:find_by).with(uid: activity.uid).and_return(activity)
+      allow(activity).to receive(:questions).and_return(questions)
+      questions.each do |question|
+        allow(question).to receive(:as_json).with(locale: locale).and_return({ "id" => question.id, "content" => "Question content" })
+      end
+    end
+
+    it "returns http success" do
+      get :index, params: { activity_id: activity.uid, locale: locale }
```

Comment:
Purely stylistic, but you can infer the value part of `locale` here: `get :index, params: { activity_id: activity.uid, locale: }`

Reply:
whoops! I think that rubocop I made is not working right...